src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -1259,6 +1261,8 @@ public LambdaInfo WithMatch(Match<SyntaxNode> match, SyntaxNode newLambdaBody)
                         if (newLambdaBody1 != null)
                         {
                             lambdaBodyMatches.Add(ComputeLambdaBodyMatch(oldLambdaBody1, newLambdaBody1, activeNodes, lazyActiveOrMatchedLambdas, diagnostics));
+
+                            ReportMethodBodySyntaxRudeEditsForLambda(GetLambda(oldLambdaBody1), GetLambda(newLambdaBody1), bodyMatch, diagnostics);
```

Comment:
>GetLambda(oldLambdaBody1) [](start = 69, length = 25)

`oldNode` and `newNode` are the nodes that represent the lambda (see the comment on `TryGetLambdaBodies`: "Returns all lambda bodies of a node representing a lambda").